cli/start.go

```diff
@@ -90,75 +64,79 @@ func (r *RootCmd) start() *clibase.Cmd {
 			return nil
 		},
 	}
-	return cmd
-}
 
-type prepStartWorkspaceArgs struct {
-	Action            WorkspaceCLIAction
-	TemplateVersionID uuid.UUID
+	cmd.Options = append(cmd.Options, parameterFlags.allOptions()...)
 
-	LastBuildParameters []codersdk.WorkspaceBuildParameter
+	return cmd
+}
 
-	PromptBuildOptions bool
-	BuildOptions       []codersdk.WorkspaceBuildParameter
+type startWorkspaceArgs struct {
```

Comment:
I'm not a fan of the "args-struct" pattern here.  It increases indirection and makes it easy to leave off required arguments.  Just make these all arguments of the function as it's only a difference of 3->5